### DISCUSSION TITLE
WebKb dataset discrepancy

### DISCUSSION THREAD

**Discussion by qubit0** [pyg-team/pytorch_geometric#4859] [Q&A] [answered]
This paper(https://arxiv.org/pdf/2002.05287.pdf) says Texas has 309 edges. However, the Texas graph from PyG has 295. Why is this so?

**Comment by rusty1s** [comment]
Thanks for reporting. I will look into it.

**Comment by rusty1s** [comment, marked as answer]
After some investigation, I believe this is a typo in the paper. You can see clearly that the official data contains 325 edges (see https://github.com/graphdml-uiuc-jlu/geom-gcn/blob/master/new_data/texas/out1_graph_edges.txt), which is exactly what PyG reports:
```
Data(x=[183, 1703], edge_index=[2, 325], y=[183], train_mask=[183, 10], val_mask=[183, 10], test_mask=[183, 10])
```

**Comment by qubit0** [comment]
Thanks for looking into it. I also thought the same.
